test/repository/sbrctrl/EditAdminDataPathSpec.scala

```diff
@@ -0,0 +1,21 @@
+package repository.sbrctrl
+
+import java.time.Month.MARCH
+
+import org.scalatest.{ FreeSpec, Matchers }
+import uk.gov.ons.sbr.models.{ Ern, Period, UnitId, UnitType }
+
+class EditAdminDataPathSpec extends FreeSpec with Matchers {
+
+  private trait Fixture {
+    val TargetId = UnitId("123456789")
```

Comment:
Not a big deal as it isn't really pertinent to this particular test, but that is not a valid VAT reference - which makes the result below a little odd ...

Reply:
Well spotted!